tests/test_manager.py

```diff
@@ -131,6 +132,23 @@ def test_ConfigIniEnv(datadir):
     assert cie.get('foo') == NO_VALUE
 
 
+def test_ConfigEnvFileEnv(datadir):
+    env_filename = os.path.join(datadir, '.env')
+    cefe = ConfigEnvFileEnv(env_filename)
+    assert cefe.get('not_a', ['youre']) == 'golfer'
```

Comment:
I've been using kwargs-style for namespace argument to make it easier to read. It's probably worth changing this line similarly.